### PR TITLE
contains fix for Updates' dialog box

### PR DESCRIPTION
**Pull request by Khushboo-5157**
contains fix for Updates' dialog box

### CODE REVIEW

File: resources/js/Layouts/AppLayout.vue [hunk 1]
```diff
@@ -487,29 +487,21 @@ watch(
...
-                                            <Link class="text-color-secondary" :href="entry.url">
-                                                (read more)
-                                            </Link>
-                                        </span>
-                                    </div>
-                                </div>
-                            </li>
-                        </ul>
+                    <div class="flex flex-column p-2">
+                        <span>We have made so many new updates that we can't fit it here, please read to see all recent updates.
```
Comment (yayann) [inline resources/js/Layouts/AppLayout.vue:499]:
```suggestion
                        
```

File: resources/js/Layouts/AppLayout.vue [hunk 2]
```diff
@@ -487,29 +487,21 @@ watch(
...
+                            <Link class="text-color-secondary" @click="visible = false" :href="route('posts.index')">
+                                [Read More]
+                            </Link>
+                        </span>
```
Comment (yayann) [inline resources/js/Layouts/AppLayout.vue:503]:
```suggestion

```

File: resources/js/Layouts/AppLayout.vue (hunk 1 above)
Comment (yayann) [reply resources/js/Layouts/AppLayout.vue:499]: Don't hardcode a message, this will be the preview text

File: resources/js/Layouts/AppLayout.vue [hunk 3]
```diff
@@ -487,29 +487,21 @@ watch(
...
-                                            </Link>
-                                        </span>
-                                    </div>
-                                </div>
-                            </li>
-                        </ul>
+                    <div class="flex flex-column p-2">
+                        <span>We have made so many new updates that we can't fit it here, please read to see all recent updates.
+                            <Link class="text-color-secondary" @click="visible = false" :href="route('posts.index')">
+                                [Read More]
```
Comment (yayann) [inline resources/js/Layouts/AppLayout.vue:501]:
```suggestion
                               <span v-html="entry.preview"></span> [Read More]
```

**Review by yayann — CHANGES_REQUESTED** [review]

File: resources/js/Layouts/AppLayout.vue (hunk 1 above)
Comment (Khushboo-5157) [reply resources/js/Layouts/AppLayout.vue]: fixed, please have a look

**Review by yayann — APPROVED** [review]

**Pull request merged**